cli/src/pcluster/config/update_policy.py

```diff
@@ -318,6 +329,8 @@ def condition_checker_managed_fsx(change, patch):
 
 
 def actions_needed_shared_storage_update(change, patch):
+    if is_home_change(change):
+        return "Please revert any changes to the /home mount"
```

Comment:
What about capture the scenario with unit tests?

Reply:
Good point, I will add these!